cafebabel/core/helpers.py

```diff
@@ -102,6 +102,20 @@ def static_pages_for(language):
     return static_pages
 
 
+def articles_for_tag(tag_slug, limit=5, exclude=None, only_published=True):
+    from ..articles.models import Article  # Circular imports.
+    from ..articles.tags.models import Tag  # Circular imports.
+    tag = Tag.objects.filter(slug=tag_slug)
+    language = current_language()
+    articles = Article.objects.filter(tags__in=tag, language=language)
+    if exclude:
+        articles = articles.filter(id__ne=exclude.id)
```

Comment:
list of ids here or is it no need?

Reply:
Actually it's not even in use anymore, I hesitate to remove it.

Reply:
At some point, it was to be able to exclude the `'editors-pick'` from subsequent tag retrieval but editors can play with tags to avoid that manually. I will remove it before merging.